Makefile

```diff
@@ -59,10 +59,11 @@ docs:
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
 
+REPO?=raiden-network/raiden
```

Comment:
Why is this parameter needed?

Reply:
It's not in normal use, but it's already supported in the Dockerfile and is sometimes useful when building manually locally.